frontend/src/components/dashboard/MineSearch.js

```diff
@@ -61,8 +60,8 @@ export class MineSearch extends Component {
    * filter mineList with new search input;
    */
   handleSearch = (value = {}) => {
-    const search = value.target && value.target.value;
-    const { commodity, region, status, tenure, tsf, major } = search ? {} : value;
+    const { commodity, region, status, tenure, tsf, major, search } = value;
+
```

Comment:
now that I think about it, we should be able to pass value in without destructuring it first

```
this.props.handleMinesearch(value)
```

and actually, may be able to delete `handleSearch` all together and pass `handleMinesearch` into the form, since we are no longer formatting the data we should no longer need this middle function.